base/server/python/pki/server/cli/cert.py

```diff
@@ -837,7 +837,7 @@ def print_help(self):  # flake8: noqa
         print()
 
     def execute(self, argv):
-        logging.basicConfig(format='%(levelname)s: %(message)s')
+        logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
```

Comment:
Do we really want to backport this logging change? Perhaps some items are marked as INFO when they shouldn't be (in the new cert-fix code?)... but I'm wondering why we're changing this to INFO here. IMO, kinda out of scope. Can we make this only apply to cert-fix and not all of the `cert` subcommands?

Reply:
what do you mean by all `cert` subcommands? This change is applies **only** for the `cert-fix`'s execute method

Reply:
Ah I missed that context. I'm fine with this then. The diff really hides the context. :/